policies/templates/gcp_vpc_sc_project_perimeter_v1.yaml

```diff
@@ -39,8 +39,8 @@ spec:
               description: "Array of service perimeters that will be allowed/denied (based on mode)
               for this project."
             project_id:
-              type: string
-              description: "The project ID you are matching"
+              type: number
```

Comment:
Are you sure this is meant to be project number and not project ID?

Reply:
yes, given the samples/*.yaml and line 83 of this template, it's definitely number.

Reply:
Got it. In that case, could we also change the field name?

Reply:
Unfortunately that's a little trickier if we already have customers using this since we would have to provide legacy support (likely in a rego workaround).

Reply:
In that case, could we add a v2 template (with these fixes) while archiving the v1 version with original name?

I'd like to avoid calling a project number a project ID in the long term. It's a very confusing UX.

Reply:
done